specification/sql/resource-manager/readme.md

```diff
@@ -63,6 +63,7 @@ input-file:
 - Microsoft.Sql/preview/2015-05-01-preview/databaseAutomaticTuning.json

```

Comment:
Please merge from master and add managedInstances and managedDatabase to ALL package-composite-v* sections.